packages/react-navigation/package.json

```diff
@@ -33,19 +33,22 @@
   "scripts": {
     "test": "jest",
     "lint": "eslint .",
-    "prepare": "bob build"
+    "prepare": "bob build",
+    "postinstall": "./fix-react-navigation-import-in-dependencies.sh"
```

Comment:
why do we need this hook in 2 places: here and in the root `package.json` and not only in one - here?

Also it means this script will be run during the package setup on the client side? If yes, are we sure that script execution is allowed on the client side and folder structure cannot be different?

Reply:
The reason that it is in the 2 places is that you can run `yarn install` from both places. If you run it at the root level, it will install all the dependencies for all the packages. If you run it here, it will only install dependencies from this package. 
So there could be an external contributor working only on this package that might run yarn from this package.

This script would not be ran if the client installed the lib as a dependency, so no impact for them.
Actually this script would be ran if the client used `npm install -g <pkg_name>`:  https://docs.npmjs.com/cli/v8/using-npm/scripts#npm-install - but there is no use case where a client would do that, that is mostly if you want to use packages that come with a cli.